boards/weact/usb2canfdv1/doc/index.rst

```diff
@@ -0,0 +1,65 @@
+.. _usb2canfdv1:
+
+WeAct Studio USB2CANFDV1
+#########################
+
+Overview
+********
+
+The WeAct Studio USB2CANFDV1 is a dedicated USB to CAN FD adapter board. More information can be
+found on the `USB2CANFDV1 website`_.
+
```

Comment:
no picture?

Reply:
I can add one from https://github.com/WeActStudio/WeActStudio.USB2CANFDV1 but I am not sure if it brings us much.

Reply:
A webp version of https://github.com/WeActStudio/WeActStudio.USB2CANFDV1/blob/master/Images%2FSWD.jpg would be nice (name it usb2canfdv1.webp). No need to actually include it in the document -- the new directive, it will be automatically pulled in for illustration purposes in the board catalog and in the new Wikipedia-like info card.

Reply:
Neat, I'll add that. How do I add an attribution for the image when included like this?

Reply:
AFAICT our illustrative use of board images falls under fair use so attribution shouldn't be required, but certainly nice to have. In fact, we prob need to start thinking where "extended" board metadata such as "datasheet url", "image attribution", "useful links", ... should live. I think it made sense to add "full_name" to board.yml but I am not convinced that's necessarily the best home for the others as it's really ever going to be used in docs, not in the build system per se. So I'm thinking they'd be arguments to the new zephyr:board directive

Reply:
Ok, I've added a .webp image as described above.